geomesa-accumulo/geomesa-accumulo-tools/src/main/resources/org/locationtech/geomesa/accumulo/tools/ingest-libjars.list

```diff
@@ -1,46 +0,0 @@
-accumulo-core
```

Comment:
How is the classpath for distributed ingest handled now?

Reply:
The main file is [here](https://github.com/elahrvivaz/geomesa/blob/fcr_distributed_export/geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportCommand.scala#L55) and then each back-end has a supplemental file aka [here](https://github.com/elahrvivaz/geomesa/blob/fcr_distributed_export/geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/AccumuloDataStoreCommand.scala#L61-L62) and [here](https://github.com/elahrvivaz/geomesa/blob/fcr_distributed_export/geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/accumulo/tools/export/AccumuloExportCommand.scala#L24)